packages/main/src/Table.js

```diff
@@ -496,6 +523,82 @@ class Table extends UI5Element {
 		}
 	}
 
+	_onkeydown(event) {
+		if (isCtrlA(event)) {
+			event.preventDefault();
+			this.isMultiSelect && this._selectAll(event);
+			return;
+		}
+
+		const isAltUp = isUpAlt(event);
+
+		if (isAltUp || isDownAlt(event)) {
+			return this._handleArrowAlt(isAltUp, event.target);
+		}
+	}
+
+	/**
+	 * Handles Alt + Up/Down.
+	 * Switches focus between column header, last focused item, and "More" button (if applicable).
+	 * @private
+	 * @param {boolean} shouldMoveUp Whether to move focus upward
+	 * @param {object} focusedElement The element currently in focus
+	 */
+	_handleArrowAlt(shouldMoveUp, focusedElement) {
+		const focusedElementType = this.getFocusedElementType(focusedElement);
+		const moreButton = this.getMoreButton();
+
+		if (shouldMoveUp) {
+			switch (focusedElementType) {
+			case "tableRow":
+				this._previousItem = focusedElement;
```

Comment:
Please name this reference as _prevFocusedItem
Previous item is not enough, it is more suitable to reference the item before the currently active (focused) one

Reply:
Done

Reply:
Done

Reply:
I would even call it _previousFocusedElement, as it stores the "focused**Element**" not the "focused**Item**".
When someone not familiar with this code reads it, he/she might misinterpret and think that this indeed stores only the items, when in reality, it stores the element the focus was last on.

Reply:
Since this reference always stores a **table row** `_prev(ious)FocusedRow` may be the most optimal. Should it be prev- or previous?

Reply:
Renamed to `_prevFocusedRow`.

Reply:
_prev or _previousFocusedRow sounds fine to me